src/WorkspaceDependencyViewExtension/WorkspaceDependencyView.xaml.cs

```diff
@@ -126,6 +134,37 @@ internal void DependencyRegen(WorkspaceModel ws)
         {
             RestartBanner.Visibility = Visibility.Hidden;
             var packageDependencies = ws.NodeLibraryDependencies.Where(d => d is PackageDependencyInfo).ToList();
+            var localDefinitions = ws.NodeLocalDefinitions.Where(d => d is LocalDefinitionInfo).ToList();
+
+            foreach (LocalDefinitionInfo info in localDefinitions)
+            {
+                try
+                {
+                    if (info.ReferenceType == ReferenceType.DYFFILE)
+                    {
+                        // Try to get the Custom node information if possible. 
+                        string customNodeName = info.Name.Replace(customNodeExtension, "");
+                        dependencyViewExtension.DependencyView.CustomNodeManager.TryGetNodeInfo(customNodeName, out CustomNodeInfo customNodeInfo);
+
+                        if (customNodeInfo != null)
+                        {
+                            info.Path = customNodeInfo.Path;
+                        }
+                    }
+
+                    if (info.Path != null)
+                    {
+                        FileInfo localDefinitionFileInfo = new FileInfo(info.Path);
+                        long size = localDefinitionFileInfo.Length / 1024;
```

Comment:
should we make 1024 constant? And would this always be in kb unit?

Reply:
Yes because the Length always returns in bytes, the result will be in kb.

Reply:
Thanks!